lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1559,17 +1559,26 @@ sub _failure_reason {
 
     my @failed_modules;
     my $modules = $self->modules;
+
```

Comment:
Unrelated whitespace change I would avoid.

Reply:
I don't like when different logical parts having code on each line, so I would rather keep it. It's a matter of taste.